projects/kit/components/input-time/input-time.component.ts

```diff
@@ -57,10 +57,6 @@ export class TuiInputTimeComponent {
     });
 
     protected setValue(value: string): void {
-        const mode = this.host.timeMode();
```

Comment:
Why all this is no longer needed?

Reply:
`TuiInputTimeDirective.setValue` implements the same logic (as all deleted lines)

https://github.com/taiga-family/taiga-ui/blob/c283c56f7d10caea1d443563a27f15233eacc883/projects/kit/components/input-time/input-time.directive.ts#L121-L123